pageserver/src/metrics.rs

```diff
@@ -595,7 +604,6 @@ pub struct TimelineMetrics {
     pub compact_time_histo: StorageTimeMetrics,
     pub create_images_time_histo: StorageTimeMetrics,
     pub logical_size_histo: StorageTimeMetrics,
-    pub load_layer_map_histo: StorageTimeMetrics,
```

Comment:
This was not a real histogram, we probably need to adjust the array where all of the operations were.

Reply:
Yes, `load layer map` is still in `STORAGE_TIME_OPERATIONS` much above this line, R33.

Reply:
Hmm we did actually have the global histogram for these. Haven't understood yet the data we have there.

Reply:
Buckets are probably not very well suited for this, but we have some >100ms entries here: https://neonprod.grafana.net/d/b7a5a5e2-1276-4bb0-9e3a-b4528adb6eb6/layermap-load-times-in-prod?from=now-28d&to=now

Reply:
After re-finding these metrics and now creating dashboards, I am questioning if the metrics change should be done, or if it should be done for all of the currently StorageHistogrammed values (only keep the global histogram).

Related to the STORAGE_TIME_OPERATIONS line R33, we also don't have "initial logical size" anymore. I don't remember the PR number but then it was forgotten to remove.

Reply:
> or if it should be done for all of the currently StorageHistogrammed values (only keep the global histogram).

1. Global histogram with _many_ buckets for each of the StorageTimeMetrics.
2. info!() log observations that exceed some configurable threshold, so that
  1. we don't flood the logs with the common case but
  2. we get the tenant/timeline ids for the outliers from the log span

The nasty thing about this would be to wire up everything in the config.
Config would need to look like

```
# pageserver.toml
...
[storage_time_metrics_thresholds]
load_layer_map = '20ms'
create_images_time = '100ms'
...

[tenant_conf]
...
```

That said, the `load_layer_map_histo` seems exceptionally boring, even on prod.

Reply:
> That said, the load_layer_map_histo seems exceptionally boring, even on prod.

Turns out after load testing, it's not that simple even in staging (now `1s - 10s` -- These are rather worrying because the layer maps in load test targets should be very simple). Similarly in prod we have `0.1s - 1s` entries.